README.md

```diff
@@ -17,6 +17,26 @@ Multer adds a `body` object and a `file` or `files` object to the `request` obje
 
 Basic usage example:
 
+### HTML FORM
+
+Singe file:
+```html
+<form action="YOUR_URL" method="post" enctype="multipart/form-data">
+  <input type="file" name="avatar">
+  <input type="submit">
+</form>
+```
+
+Multiple files:
+```html
+<form action="YOUR_URL" method="post" enctype="multipart/form-data">
+  <input type="file" name="avatar" multiple>
```

Comment:
Can we change this name to `photos` instead of changing below? I think it's clearer if they use different names, and it also helps that "photos" is plural and "avatar" is singular